openprescribing/dmd/management/commands/bulk_import_drug_tariff.py

```diff
@@ -0,0 +1,72 @@
+"""Bulk importer for manually-prepared tariff CSV.
+
+
+This probably won't be used again following initial data load, so
+could be deleted after that.
+
+"""
+import csv
+import logging
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from dmd.models import TariffPrice
+from dmd.models import DMDVmpp
+from dmd.models import DMDProduct
+from frontend.models import ImportLog
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    args = ''
+    help = 'Imports a CSV of historic tariff prices'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--csv')
+
+    def handle(self, *args, **options):
+        '''
+        '''
+        with open(options['csv'], 'r') as f:
+            with transaction.atomic():
+                month = None
+                counter = 0
+                for row in csv.DictReader(f):
+                    month = datetime.strptime(row['Month'], '%d/%m/%Y')
+                    counter += 1
+                    if 'Category A' in row['DT Cat']:
+                        tariff_category = 1
+                    elif 'Category C' in row['DT Cat']:
+                        tariff_category = 3
+                    elif 'Category M' in row['DT Cat']:
+                        tariff_category = 11
+                    else:
+                        raise
```

Comment:
A bare `raise` isn't quite valid Python -- it'll actually raise a `TypeError`.  (You can take the coder out of Ruby, etc etc.)  It's harmless here, but a pattern I'm finding myself using is `assert False, msg`, where `msg` is something useful -- in this case, probably the value of `row['DT Cat']`.

Reply:
Nice!